activerecord/lib/active_record/connection_adapters/abstract/quoting.rb

```diff
@@ -130,6 +130,7 @@ def quoted_date(value)
       end
 
       def quoted_time(value) # :nodoc:
+        value = value.change(year: 2000, month: 1, day: 1)
         quoted_date(value).sub(/\A\d\d\d\d-\d\d-\d\d /, "")
```

Comment:
If we know the date is 2001-01-01 we can probably just strip off the first 11 characters and save ourselves the `sub`